Checked grammar all the way #1632 (#1697)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/safeguarding-your-data.md

---

# How SharePoint Online and OneDrive safeguard your data in the cloud
 
You control your data. When you put your data in SharePoint Online and OneDrive, you remain the owner of the data. For more info about the ownership of your data, see [Office 365 Privacy by Design](https://www.microsoft.com/trustcenter/privacy/privacy-overview).
  
## How we treat your data
 
Microsoft engineers administer SharePoint Online and OneDrive using a Windows PowerShell console that requires two-factor authentication. We perform day-to-day tasks by running workflows so we can rapidly respond to new situations. Check-ins to the service require code review and management approval.
  

> No engineer has standing access to the service. When engineers need access, they must request it. Eligibility is checked, and if engineer access is approved, it's only for a limited time. In rare cases where Microsoft engineers need access to content (for example, if you submit a support ticket because a user can't access an important file that we believe is damaged), the engineers must check in a specific workflow that requires business justification and manager approval. An audit event is generated that you can view in the Microsoft 365 admin center. You can also turn on a feature called Customer Lockbox, so you need to approve. The engineer gets access only to the file in question. Learn how to turn on or off Customer Lockbox and approve and deny requests: [Office 365 Customer Lockbox Requests](/office365/admin/manage/customer-lockbox-requests).

Reviewer: ```suggestion
No engineer has standing access to the service. When engineers need access, they must request it. Eligibility is checked, and if engineer access is approved, it's only for a limited time. In rare cases where Microsoft engineers need access to content (for example, if you submit a support ticket because a user can't access an important file that we believe is damaged), the engineers must check in a specific workflow that requires business justification and manager approval. An audit event is generated that you can view in the Microsoft 365 admin center. You can also turn on a feature called Customer Lockbox, so you need to approve the request. The engineer gets access only to the file in question. Learn how to turn on or off Customer Lockbox and approve and deny requests: [Office 365 Customer Lockbox Requests](/office365/admin/manage/customer-lockbox-requests).
```

- Prevent accidental exposure of sensitive content. Create DLP policies to identify documents and prevent them from being shared. See [Overview of data loss prevention policies](/office365/securitycompliance/data-loss-prevention-policies).
    
## Protected in transit and at rest
 
### Protected in transit
 

> When data transits into the service from clients, and between data centers, it's protected using best-in-class encryption. For info, see [Data Encryption in OneDrive for Business and SharePoint Online](/office365/securitycompliance/data-encryption-in-odb-and-spo). We only permit secure access. We won't make authenticated connections over HTTP but, instead, redirect to HTTPS.

Reviewer: ```suggestion
When data transits into the service from clients, and between datacenters, it's protected using best-in-class encryption. For info, see [Data Encryption in OneDrive for Business and SharePoint Online](/office365/securitycompliance/data-encryption-in-odb-and-spo). We only permit secure access. We won't make authenticated connections over HTTP but, instead, redirect to HTTPS.
```

  
### Protected at rest
 

>  **Physical protection**: Only a limited number of essential personnel can gain access to datacenters. Their identities are verified with multiple factors of authentication, including smart cards and biometrics. There are on-premises security officers, motion sensors, and video surveillance. Intrusion detection alerts monitor anomalous activity. 

Reviewer: ```suggestion
 **Physical protection**: Only a limited number of essential personnel can gain access to data centers. Their identities are verified with multiple factors of authentication, including smart cards and biometrics. There are on-premises security officers, motion sensors, and video surveillance. Intrusion detection alerts monitor anomalous activity. 
```

  
- Configure IRM policies on SharePoint document libraries to limit download of content. See [Set up Information Rights Management (IRM) in SharePoint admin center](/office365/securitycompliance/set-up-irm-in-sp-admin-center).
    
- Evaluate the use of Azure Information Protection (AIP). Classification and labeling let you track and control how data is used. Visit [Azure Information Protection](https://azure.microsoft.com/services/information-protection/).
    
## Highly available, always recoverable
 

> Our datacenters are geo-distributed within the region and fault-tolerant. Data is mirrored in at least two datacenters to mitigate the impact of a natural disaster or service-impacting outage. See the [Where's my data?](https://o365datacentermap.azurewebsites.net/).

Reviewer: ```suggestion
Our datacenters are geo-distributed within the region and fault tolerant. Data is mirrored in at least two datacenters to mitigate the impact of a natural disaster or service-impacting outage. For more information, see [Where's my data?](https://o365datacentermap.azurewebsites.net/).
```

  
Metadata backups are kept for 14 days and can be restored to any point in time within a 5-minute window. 
  

> In the case of a ransomware attack, you can use Version history ([Enable and configure versioning for a list or library](https://support.office.com/article/1555d642-23ee-446a-990a-bcab618c7a37)) to roll back, and the recycle bin or site collection recycle bin to restore ([Restore deleted items from the site collection recycle bin ](https://support.office.com/article/5fa924ee-16d7-487b-9a0a-021b9062d14b)). If an item is removed from the site collection recycle bin, you can call support within 14 days to access a backup. For info about the new Files Restore feature that lets users restore an entire OneDrive to any point within the past 30 days, see [Restore your OneDrive](https://support.office.com/article/fa231298-759d-41cf-bcd0-25ac53eb8a15.aspx).

Reviewer: ```suggestion
In the case of a ransomware attack, you can use Version history ([Enable and configure versioning for a list or library](https://support.office.com/article/1555d642-23ee-446a-990a-bcab618c7a37)) to roll back, and the recycle bin or site collection recycle bin to restore ([Restore deleted items from the site collection recycle bin](https://support.office.com/article/5fa924ee-16d7-487b-9a0a-021b9062d14b)). If an item is removed from the site collection recycle bin, you can call support within 14 days to access a backup. For information about the new Files Restore feature that lets users restore an entire OneDrive to any point within the past 30 days, see [Restore your OneDrive](https://support.office.com/article/fa231298-759d-41cf-bcd0-25ac53eb8a15.aspx).
```
